code_examples/sdk_examples/src/test.ts

```diff
@@ -0,0 +1,90 @@
+import * as Kilt from '@kiltprotocol/sdk-js'
+import { config as envConfig } from 'dotenv'
+import { hexToU8a } from '@polkadot/util'
+import { program } from 'commander'
+import { testCoreFeatures } from './core_features'
+import { testDapp } from './dapp'
+import { testStaking } from './staking'
+import { testWorkshop } from './workshop'
+
+const SEED_ENV = 'FAUCET_SEED'
+
+;(async () => {
+  const whichToRun = {
+    workshop: false,
+    dapp: false,
+    core: false,
+    staking: false
+  }
+  program.description('Test the code examples used in the KILT documentation.')
+  program
+    .command('all', { isDefault: true })
+    .description('Run all tests')
+    .action(() => {
+      for (const key in whichToRun) {
+        whichToRun[key] = true
+      }
+    })
+  program
+    .command('workshop')
+    .description('Test code examples inside the workshop')
+    .action(() => {
+      whichToRun.workshop = true
+    })
+  program
+    .command('dapp')
+    .description('Test code examples inside the DApp section')
+    .action(() => {
+      whichToRun.dapp = true
+    })
+  program
+    .command('core')
+    .description('Test code examples inside the Core Feature section')
+    .action(() => {
+      whichToRun.core = true
+    })
+  program
+    .command('staking')
+    .description('Test code examples inside the Staking section')
+    .action(() => {
+      whichToRun.staking = true
+    })
+  program.parse()
+
+  envConfig()
+  await Kilt.init()
+
+  const wssAddress =
+    process.env.WSS_ADDRESS || 'wss://peregrine.kilt.io/parachain-public-ws'
+  const faucetSeed = process.env[SEED_ENV]
+
+  if (!faucetSeed) {
+    console.log(
+      `Account seed with sufficient balance is required. Set the secret seed using the ${SEED_ENV} environment variable.`
+    )
+    throw 'Account seed is missing'
```

Comment:
probably doesn't matter, but you are throwing a string here (yes, you can do that in js). If you want to throw an error, you have to be explicit
```suggestion
    throw new Error('Account seed is missing')
```

Reply:
[167f846](https://github.com/KILTprotocol/docs/pull/197/commits/167f84609ed23dc27a287ab6baa3392317ec952d)